README.md

```diff
@@ -223,8 +223,79 @@ val retrofit = Retrofit.Builder().build("your_baseUrl", okHttpClient gsonConvert
 </li>
 </ul>
 
+### Event Logger
+
+A remote troubleshooting utility which helps to track or send important events to a remote API server.
+This is intended to be used internally by Rakuten's SDKs.
+
+<ul>
+<li>
+
+```configure```: Sets the server configuration and other initialization processing. Call this as early as possible in the application lifecycle, such as `onCreate` or other initialization methods. Calling other APIs without calling this API has no effect.
+
+| Parameter | Description              | Datatype | Required? |
+|-----------| ------------------------ |----------|-----------|
+| context   | Application context      | Context  | `Yes`     |
+| apiUrl    | Non-empty server API URL | String   | `Yes`     |
+| apiKey    | Non-empty server API Key | String   | `Yes`     |
```

Comment:
I think we should also mention how can application get/ generate these **apiUrl**  and **apiKey** parameters.

Reply:
Theoretically this should be any value as long as it supports the EventLogger /error endpoint.
It will be provided via RMC, and since sdk-utils does not have context of RMC, it wasn't mentioned in this guide.

For now will keep it as it is, same as how these configurations are mentioned in IAM SDK guide:
https://rakutentech.github.io/android-inappmessaging/docs/7.5/#:~:text=%234%20Adding%20subscription%20ID%20and%20config%20URL.

If needed will update this guide.